overlord/servicestate/quota_handlers.go

```diff
@@ -793,3 +897,25 @@ func quotaControlAffectedSnaps(t *state.Task) (snaps []string, err error) {
 	}
 	return snaps, nil
 }
+
+func quotaAddSnapAffectedQuotas(t *state.Task) (quotas []string, err error) {
```

Comment:
unit tests don't seem to trigger this code

Reply:
Fixed!